docs/time_series_for_support_staff.md

```diff
@@ -0,0 +1,159 @@
+
+---
+
+`riak-shell` The Developer/Sys-Admin's New Best Pal
+---------------------------------------------------
+
+The `riak-shell` is a new way of interacting with a riak cluster - it lets you:
+
+* create time series tables 
+* insert data into those tables
+* query data from those tables
+
+---
+
+`riak-shell` Logging And Replay
+-------------------------------
+
+In addition `riak-shell` can be set up to log your commands and replay those logs. This lets you prepare tables for you application in development environments, capture the configuration in text files which can be placed under version control and use those scripts to set up your Integration, UAT and Production environments.
+
+The log facility can also be used to report bugs to Basho. Find a problem, log it happening from the shell and send us the log as a bug report with notes of how it should work. This enables Basho to reproduce your problem.
+
+The replay facility of `riak-shell` also lets us capture your bug straight into our regression test suite - we can kill it for you and ensure that it stays killed in all future releases.
+
+Please read the `riak-shell` documentation for more details:
+
+* [`riak-shell` Documentation](https://github.com/basho/riak_shell/blob/develop/README.md)
+
+---
+
+Data On Disk
+------------
+
+When a Time Series table is created in a riak cluster a dynamic module is generated and loaded in to the Erlang VM. This module is stored in the `data/ddl_ebin` directory alongside the `data/kv_vnode` and other data directories. The state of the compilation and the definitive list of which DDLs have been created and associated information is stored in an Erlang dets table `data/riak_kv_compile_tab.dets`
+
+These helper modules enable the riak cluster to introspect Time Series data on the fly. By default no source code for them is generated - only `.beam` files. If you wish to see what they look like *under the covers* the riak_ql [`README`](https://github.com/basho/riak_ql/blob/develop/README.md) has an example of one.
+
+---
+
+FAQ 1
+-----
+
+**Q:** What tables have been created?
+
+**A:** list the files in `data/ddl_ebin` - they will have names like `riak_ql_table_`**TABLE NAME**`$1.beam`
+
+---
+
+FAQ 2
+-----
+
+**Q:** How do I find out the definition of a table that has been created?
+
+**A:** there are 2 ways to do this:
+
+First method:
+
+connect to the riak node with `riak attach` in `ebin/` and run:
+
+`1>DDL = 'riak_ql_table_my_table_name$1':get_dll().`
+
+**Note**: the module name must be a quoted atoms because it contains a `$` symbol
+
+You can convert that table representation to a string with a helper function:
+
+`2>riak_ql_to_string:ddl_rec_to_SQL(DDL).`
+
+---
+
+FAQ 2 (cont)
+------------
+
+**Q:** How do I find out the definition if a table that has been created?
+
+**A:** there are 2 ways to do this:
+
+Second method:
+
+connect to the shell by running `riak-shell` in `ebin/` and run:
```

Comment:
Swap these two. Clearly `riak-shell` is easier and less dangerous, and some people will stop at the first method and use that.